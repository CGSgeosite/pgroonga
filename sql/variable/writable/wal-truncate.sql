SET pgroonga.enable_wal = true;

CREATE TABLE memos (
  content text
);

INSERT INTO memos VALUES ('Groonga is fast!');

CREATE INDEX pgrn_index ON memos USING PGroonga (content);

SET pgroonga.writable = false;

SELECT pgroonga_wal_truncate('pgrn_index');

SET pgroonga.writable = true;

SELECT pgroonga_wal_truncate('pgrn_index');

DROP TABLE memos;
