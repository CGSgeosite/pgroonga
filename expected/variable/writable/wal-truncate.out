SET pgroonga.enable_wal = true;
CREATE TABLE memos (
  content text
);
INSERT INTO memos VALUES ('Groonga is fast!');
CREATE INDEX pgrn_index ON memos USING PGroonga (content);
SET pgroonga.writable = false;
SELECT pgroonga_wal_truncate('pgrn_index');
 pgroonga_wal_truncate 
-----------------------
                     2
(1 row)

SET pgroonga.writable = true;
SELECT pgroonga_wal_truncate('pgrn_index');
 pgroonga_wal_truncate 
-----------------------
                     2
(1 row)

DROP TABLE memos;
